docs/examples-search-as-you-type.mdx

```diff
@@ -17,7 +17,7 @@ It's a good idea to add a debounce time — the Search UI will wait for users to
 See the example implementation below.
 
 <iframe
-  src="https://codesandbox.io/embed/github/elastic/search-ui/tree/stable/examples/sandbox?autoresize=1&fontsize=12&initialpath=%2Fsearch-as-you-type&module=%2Fsrc%2Fpages%2Fsearch-as-you-type%2Findex.js&theme=light&view=preview&hidedevtools=1"
+  src="https://codesandbox.io/embed/github/elastic/search-ui/tree/master/examples/sandbox?autoresize=1&fontsize=12&initialpath=%2Fsearch-as-you-type&module=%2Fsrc%2Fpages%2Fsearch-as-you-type%2Findex.js&theme=light&view=preview&hidedevtools=1"
```

Comment:
Switching to master to enable this demo before the npm release. It's broken on prod now.